setup.sh

```diff
@@ -25,7 +25,7 @@ install_docker(){
 create_directories() {
     if [[ ! -e $REDASH_BASE_PATH ]]; then
         sudo mkdir -p $REDASH_BASE_PATH
-        sudo chown $USER:$USER $REDASH_BASE_PATH
+        sudo chown "$USER:`id -gn`" $REDASH_BASE_PATH
```

Comment:
Hmmm, maybe go with the `chown $USER:` approach instead, as chown will default to the users primary group if `:` is included but the group name is empty.

So, that'd make it:

```
sudo chown "$USER:" $REDASH_BASE_PATH
```